### ISSUE TITLE
Support private note migration

### ISSUE THREAD

**Issue by eobet** [mastodon/mastodon#22709]
### Pitch

Followers are migrated automatically. Follows need to be imported manually. With private notes you can't do neither.

This needs to be fixed. Ideally, they should also carry over automatically (as should everything else, but that's a separate issue).

### Motivation

Migration is very perilous and a super advanced feature currently. Everything possible should be done in order to streamline it. This is just a small part.

**Comment by Cassolotl** [comment]
**Yes!**

I recently moved to a new instance, and I was able to export my blocks, but I was unable to export my notes. But I do kick people off my follower list a lot by block-unblocking (is that soft-blocking?), and I always add a note to explain to myself why I should ignore their follow request in future. So now I'm on a new account, people are requesting to follow me, they seem familiar, but there is no note - so I don't know when I should reject a follow request due to prior experience.

**Comment by velw** [comment]
Would love to see this as well. Glad to see it noted here as I've been meaning to do the same. Even if it remains a manual CSV export/import for now, it'd still be really useful.

**Comment by nie7321** [comment]
When one of my followers migrates, it would be excellent if the private note for their moved account would automatically be copied to their new account -- although I am not sure if there's enough information in the follow request to do that?

**Comment by JPLeBreton** [comment]
As far as I can tell, the private notes data also doesn't make it into a user account export "archive takeout". So that should probably be filed as a separate issue. Yet another missing floorboard in the promises of account migration and federation in general.

**Comment by ThisIsMissEm** [comment]
Have raised this on SocialHub regarding inclusion in the zip-file based account backups, since those are in AS2 vocabulary format, and we'd need to figure out the right way to structure this information: https://socialhub.activitypub.rocks/t/mechanism-for-representing-account-notes-in-actor-objects-or-relationships/4637?u=thisismissem

However, in the mean time, we could add an additional column to CSV exports of follows, blocks, and mutes, as well as a CSV export of your account notes entirely.

I asked @renchap about it on discord, who responded "sounds useful?"; The bulk of the work is in https://github.com/mastodon/mastodon/blob/d95f6f4410c28b2f7b9f736c9477936587bd700b/app/models/export.rb as well as in the import code paths: https://github.com/mastodon/mastodon/blob/11e0049b08c721eb98d8220405c2ffc16041ed17/app/controllers/settings/imports_controller.rb
